test_data/swc/strahler.swc

```diff
@@ -4,22 +4,23 @@
 # https://commons.wikimedia.org/wiki/File:Flussordnung_(Strahler).svg
 
```

Comment:
can you add a comment that describes that there is now a difference from the link above?

Reply:
You mean, with the strahler orders represented in the pictures ?
Because they haven't changed. Actually, if I am not mistaken, the file is now more morphologically correct than it used to be because on the picture the branch is not starting with a bifurcation.